x-pack/plugins/viz_editor/public/common/components/operation_editor/operation_definitions.tsx

```diff
@@ -0,0 +1,621 @@
+/*
```

Comment:
Meta comment on this entire file:  A big chunk of this is defining reusable logic for operations that will presumably be used outside of components. I would expect that the components would reuse existing logic from `lib`, or that you would extend `lib` to keep the react components as small as possible

Reply:
You are right, it makes sense to split this up. However I would move this into a different PR or even just note this down somewhere to do better in our clean setup.